python/ray/serve/_private/proxy.py

```diff
@@ -1510,22 +1356,6 @@ async def _uvicorn_keep_alive(self) -> Optional[int]:
             return self._uvicorn_server.config.timeout_keep_alive
 
 
-def _determine_target_loop():
```

Comment:
this wasn't doing anything because ray already set up the event loop by the time it runs